packages/core/src/_test/ens/app.test.ts

```diff
@@ -53,7 +53,7 @@ const setup = async ({ context }: { context: TestContext }) => {
 
   const gql = async (query: string) => {
     const response = await request(ponder.serverService.app)
-      .post("/")
+      .post("/graphql")
```

Comment:
fixes this:

https://github.com/0xOlias/ponder/actions/runs/6388656128/job/17338850870?pr=367#step:5:103